rules/framework/vfs_overlay.bzl

```
@@ -57,7 +58,18 @@ def _build_subtrees(paths, vfs_prefix):
     for path_info in paths:
         path = path_info.framework_path
 
-        parts = path.split("/")
+        # Avoid calling `.split` when possible for better performance
+        parts = []
+        parts_len = 0
+        if "/" in path:
+            parts = path.split("/")
+            parts_len = len(parts)
+        else:
+            parts = [path]
+            parts_len = 1
+
+        if parts_len == 0:
+            fail("[ERROR] Failed to build VFS subtrees, path with empty split: parts=%s, parts_len=%s" % (parts, parts_len))
```

```suggestion
            fail("[ERROR] Failed to build VFS subtrees, path with empty split: parts=%s" % parts)
```

nit: parts_len seems redundant since it's only ever the case this is == 0

---

nit: a better error message would print the original `path` and say it was trying to split on `/`

---

SGTM! Will update.

---

This is just me being paranoid in case someone uses `parts_len` in a way that was not intended, regardless makes sense to improve the err msg: https://github.com/bazel-ios/rules_ios/pull/772/commits/8a662017dc7a2a888dcfeebaf7a0d0c85c891b42